### ISSUE TITLE
Seek new maintainers, fork into Typelevel github organization and publish under org.typelevel

### ISSUE THREAD

**Issue by milessabin** [typelevel/squants#139]
Following up on #137, this is a call for new maintainers under the [typelevel github organization](https://github.com/typelevel). Other than @garyKeorkunian the current contributors are listed as,

* Jeremy Apthorp ([nornagon](https://github.com/nornagon))
* Steve Barham ([stevebarham](https://github.com/stevebarham))
* Derek Morr ([derekmorr](https://github.com/derekmorr))
* rmihael ([rmihael](https://github.com/rmihael))
* Florian Nussberger ([fnussber](https://github.com/fnussber))
* Ajay Chandran ([ajaychandran](https://github.com/ajaychandran))
* Gia Bảo ([giabao](https://github.com/giabao))
* Josh Lemer ([joshlemer](https://github.com/joshlemer))
* Dave DeCarpio ([DaveDeCaprio](https://github.com/DaveDeCaprio))
* Carlos Quiroz ([cquiroz](https://github.com/cquiroz))

and it'd be good to see one or more of them step forward. I imagine that everyone who has commented on #137 is also a stakeholder in some sense so they would be very welcome to step forward too.

Given that @garyKeorkunian approached Typelevel for member status I think it's appropriate to continue the project in the Typelevel organization. That being the case I think it would be good if volunteer maintainers shared that commitment.

**Comment by cquiroz** [comment]
Hi, I'd be happy to keep squants under the typelevel umbrella

**Comment by darkfrog26** [comment]
I've already volunteered myself, so I guess I'll do so again here.

**Comment by vitorsvieira** [comment]
Me too!

**Comment by golem131** [comment]
Plus me

**Comment by milessabin** [comment]
@sarahgerweck you were the one who broached the topic of new maintainership ... would you like to volunteer as well?

**Comment by golem131** [comment]
@milessabin what's next?

**Comment by milessabin** [comment]
1. I fork.
2. I add the people volunteering here as contributors.
3. We arrange for one of the above to be able to publish to Sonatype/Central under the "org.typelevel" organization.

On (3) I propose that person be @cquiroz if that's acceptable to everyone here.

**Comment by sarahgerweck** [comment]
@milessabin I haven't worked on the code at all but I've liked this library, so yeah I'm willing to be on the list if we're setting up a pool of maintainers in @garyKeorkunian's absence. I concur that @cquiroz makes sense as the lead maintainer since he's a previous contributor.

**Comment by milessabin** [comment]
Given that @garyKeorkunian has just merged #137 I'm going to hold off until he comments here.

**Comment by garyKeorkunian** [comment]
Hello All,

The artifacts have been published and so you should all be able to begin using the 0.7.x-SNAPSHOT for Scala 2.12.  Updates to the 0.6 Release may still be required if anyone needs them.

I've added a couple of folks to the collaborator list and will others listed here.  

Sounds like the best approach going forward is to move the project to Typelevel.  I support that, and Miles, you can add me as a collaborator as well.  As I mentioned on the other thread, at my new gig we don't use Squants (no use case for quantities), so my focus on this has been diverted.  Nevertheless, I am happy to do reviews and make other contributions when I can.

Again, I apologize to everyone for blocking your progress.  It is nice to see the adoption, though.  It's truly outgrown me.  

Thanks everyone!

**Comment by golem131** [comment]
Final, some questions:
1) who(except @garyKeorkunian) can publish artifacts to sonatype?
2) we have - #128 PR, what with him?
3) what is new milestone (0.7 or 1.0)?

**Comment by golem131** [comment]
@milessabin or @garyKeorkunian please resend me invitation to collaborators (old is not active now)

**Comment by milessabin** [comment]
@golem131 invites to you and @sarahgerweck have been resent.

**Comment by milessabin** [comment]
Currently the project is still using the "com.squants" artefact organization, so the only people able to publish are people with the relevant permissions for doing that.

If you would like to change that to "org.typelevel" you'll need to change your build to reflect that. I also recommend making a large enough version bump to indicate a breaking change in whatever version numbering scheme you choose.

Ping me if/when you've done that and I'll start the process of granting @cquiroz the permissions needed to publish under "org.typelevel".

**Comment by cquiroz** [comment]
@milessabin Could you summarize who has commiter rights now?

I think it makes sense to change to `org.typelevel` and probably version to 1.0 to reflect the change

**Comment by milessabin** [comment]
@cquiroz I think I just did ... whoever can publish as "com.squants".

**Comment by cquiroz** [comment]
I think this is pretty much settled. This is the current list of committers shown on the project:

@vitorsvieira
@darkfrog26
@cquiroz (admin)
@derekmorr
@milessabin
@garyKeorkunian (admin)
@golem131

Plus one invite awaiting confirmation

If you think your name is missing just let us know

**Comment by cquiroz** [comment]
I think, this can be closed, reopen if needed
